_posts/theme/2021-02-04-Scrolls.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: theme
+title: scrolls
+category: theme
+date: 2021-02-04
+homepage: https://github.com/daodaolee/typora-scrolls
+download: https://github.com/daodaolee/typora-scrolls/archive/0.2.zip
+built-in: false
+author: DaoDaoLee
+thumbnail: thumbnail
```

Comment:
@daodaolee here, thumbnail should be "scrolls.png", or we cannot display the thumbnail

Reply:
ok，i edit it right now